pkg/controller/operandbindinfo/operandbindinfo_controller.go

```diff
@@ -139,17 +140,31 @@ func (r *ReconcileOperandBindInfo) Reconcile(request reconcile.Request) (reconci
 	if len(requestNamespaces) == 0 {
 		return reconcile.Result{}, nil
 	}
+	// Get the operand namespace
+	var operandNamespace string
+	for _, op := range registryInstance.Spec.Operators {
+		if op.Name == bindInfoInstance.Spec.Operand {
+			operandNamespace = op.Namespace
```

Comment:
Add a `break` is better when the namespace found

Reply:
Will add

Reply:
@DanielXLee Done